api/organization/get.js

```diff
@@ -28,6 +24,16 @@ module.exports = async (req, res, ctx) => {
       snapshots: org.snapshots
     }
 
+    // if the caller doesn't care about auth'd org data, get out quick
+    if (noAuth) {
```

Comment:
I added this flag because I noticed we use the "is this user a github admin" call A LOT and it's SLOW AF. for some calls, we don't care at all (like on the ledger page, we only care about the org's name!). also... sometimes we get rate limited on the admin-check from GH, just by refreshing the UI a couple of times. so we should noodle this and maybe cache it, idk.